src/Neighbor/NodePairList.hh

```diff
@@ -22,7 +25,7 @@ public:
 
   NodePairList()                                                               = default;
   NodePairList(const NodePairList& rhs)                                        = default;
-  ~NodePairList()                                                              = default;
+  ~NodePairList() { mManagedNPL.free(); }
```

Comment:
To stylistically match the other lines in this file this inline body should be shifted to the right to match the surrounding lines.